### PR TITLE
gi-docgen: 2021.2 → 2021.5

### PR DESCRIPTION
**Pull request by jtojnar**
###### Things done

- [x] Docs of all packages built by this look fine
- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 122919` at dc0f2efd run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>787 packages marked as broken and skipped:</summary>
  <ul>
    <li>adobe-reader</li>
    <li>adoptopenjdk-hotspot-bin-13</li>
    <li>adoptopenjdk-hotspot-bin-14</li>
    <li>adoptopenjdk-jre-hotspot-bin-13</li>
    <li>adoptopenjdk-jre-hotspot-bin-14</li>
    <li>adoptopenjdk-jre-openj9-bin-13</li>
    <li>adoptopenjdk-jre-openj9-bin-14</li>
    <li>adoptopenjdk-openj9-bin-13</li>
    <li>adoptopenjdk-openj9-bin-14</li>
    <li>agdaPackages.iowa-stdlib</li>
    <li>...</li>
  </ul>
</details>
<details>
  <summary>14332 packages skipped due to time constraints:</summary>
  <ul>
    <li>AusweisApp2</li>
    <li>DisnixWebService</li>
    <li>EBTKS</li>
    <li>Fabric (python38Packages.Fabric)</li>
    <li>MIDIVisualizer</li>
    <li>OSCAR</li>
    <li>R</li>
    <li>SDL_Pango</li>
    <li>SDL_mixer</li>
    <li>Sylk</li>
    <li>...</li>
  </ul>
</details>
<details>
  <summary>6 packages built successfully:</summary>
  <ul>
    <li>asciidoctor</li>
    <li>gdk-pixbuf</li>
    <li>gi-docgen</li>
    <li>pango (gnome2.pango)</li>
    <li>lasem</li>
    <li>libnotify</li>
  </ul>
</details>

File: pkgs/development/tools/documentation/gi-docgen/default.nix [hunk 1]
```diff
@@ -1,75 +1,24 @@
...
   pname = "gi-docgen";
-  version = "2021.2";
+  version = "2021.5";
 
   format = "other";
 
   src = fetchFromGitLab {
     domain = "gitlab.gnome.org";
-    owner = "ebassi";
+    owner = "GNOME";
```
Comment (dasj19) [inline pkgs/development/tools/documentation/gi-docgen/default.nix:16]: What is the reason for changing to ebassi's repo? Doesn't he make merge requests to the main GNOME repo?

Comment (jtojnar) [reply pkgs/development/tools/documentation/gi-docgen/default.nix:16]: It is the other way around. It was moved from a private repo to GNOME namespace.

Comment (dasj19) [reply pkgs/development/tools/documentation/gi-docgen/default.nix:16]: Oh sorry, my bad, I read it wrongly. Then it's all good

**Pull request merged**